plugins/woocommerce-admin/client/task-lists/components/test/task-list-item.test.tsx

```diff
@@ -174,6 +188,25 @@ describe( 'TaskListItem', () => {
 		);
 	} );
 
+	it( 'should call trackClick and trigger recordEvent', () => {
```

Comment:
What is `trackClick` referring to here?

Reply:
Good question! I was referring to the function [here](https://github.com/woocommerce/woocommerce/blob/d8199e7cc3b18f8df5e55540dfde0146c914a0ef/plugins/woocommerce-admin/client/task-lists/components/task-list-item.tsx#L133), but looking at it again it actually is confusing since it doesn't have the context. I'll change the test name a bit

Reply:
Updated in [e778537](https://github.com/woocommerce/woocommerce/pull/47876/commits/e778537afa3cf05fa0dc7844df4d219d1d58c783)